resources/energyplus/ProposedEnergy+.idd

```diff
@@ -30845,7 +30849,7 @@ Coil:Cooling:DX:TwoSpeed,
        \minimum 0.0
        \maximum 1250.0
        \default 773.3
-  N13, \field Low Speed 2023 Rated Evaporator Fan Power Per Volume Flow Rate
+  N13, \field Low Speed Rated Evaporator Fan Power Per Volume Flow Rate 2023
```

Comment:
This is the approach we've taken across all object/fields, right?

Reply:
Yeah the reason is dump, but this is because of our parser...

"Speed 1 field 2023" -> 1 is removed, 2023 is kept
"Speed 2023 field" -> 2023 is removed